### DISCUSSION TITLE
Package is not working?

### DISCUSSION THREAD

**Discussion by anonymouse-cmd** [SwiftyProgrammer690/fireworks#4] [Q&A] [answered]
So what is happening to me is that even though I have cloned the necessary files and I open a python file fresh and decide to try this, it gives me errors...
```
Traceback (most recent call last):
  File "main.py", line 3, in <module>
    get_org_repositories("Microsoft")
NameError: name 'get_org_repositories' is not defined
```
This is my code:
`get_org_repositories("Microsoft")`

**Comment by SwiftyProgrammer690** [comment, marked as answer]
Hmm... Seems like you forgot to put this very important line of code at the top: `from src.source import *`
That should work now!

**Reply by anonymouse-cmd** [reply]
Now it works! Thank you!
